src/Finelines/Stages/YamlStage.fs

```diff
@@ -21,6 +22,10 @@ type YamlStage with
             this.DisplayName
             |> Option.map (fun name -> $"displayName: {format (Text name)}")
 
+        let condition = 
```

Comment:
One problem with using `Text` is that the result gets wrapped in single quotes, which produces invalid yaml:
```
condition: 'and(succeeded(), in(variables['Build.Reason'], 'IndividualCI'))'
```

My guess is that we would like to extend the `Input` DU with a `Condition` or `Expression` or `ConditionExpression` case, that would never wrap the string value in quotes in `Format.fs`, and use it here:
```
let condition = 
    this.Condition
    |> Option.map (fun condition -> $"condition: {format (ConditionExpression condition)}")
```

Reply:
> One problem with using Text is that the result gets wrapped in single quotes, which produces invalid yaml if the expression already contains single quotes:

Yes, correct, in general we need a proper escaping mechanism as YAML quotes are [very evil](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml). What you suggest is definitely one direction to go.

> What is the best way to contribute to this project? A fork and PRs as usual?

Sure, as usual. GitHub flow :)

Reply:
Escaping makes sense. Not sure about conditions specifically though, since all the examples I see in the docs don't use quotes around conditions. Are there any cases where it would not work without quotes?
I've added a commit and opened a PR into yours meanwhile - please take a look 🙂 
https://github.com/psfinaki/Finelines/pull/5

Reply:
> Are there any cases where it would not work without quotes?

Not that I know of, but I don't think we need to be over paranoid about this so that we don't complicate the code too much right away. If any case appears, we'll elaborate the logic :)

Reply:
I've a couple of unit tests for the new DU case as well. Feel free to re-review then!

Reply:
Awesome! 🙂